pkg/menu/menu.go

```diff
@@ -0,0 +1,72 @@
+package menu
+
+import (
+	"io"
+
+	ui "github.com/gizak/termui/v3"
+	"github.com/gizak/termui/v3/widgets"
+)
+
+const menuWidth = 50
+
+// newParagraph will return a paragraph object with given initial text.
+func newParagraph(initText string, border bool, location int, wid int, ht int) *widgets.Paragraph {
+	p := widgets.NewParagraph()
+	p.Text = initText
+	p.Border = border
+	p.SetRect(0, location, wid, location+ht)
+	p.TextStyle.Fg = ui.ColorWhite
+	ui.Render(p)
```

Comment:
In general, the UI packages I have seen separate the creation of a thing from rendering that thing. So a newParagraph creates the new paragraph with all the options, but another function would normally render it. This is just a thought and it is up to you.